web/src/app/models/loi.model.ts

```diff
@@ -86,4 +27,12 @@ export class GenericLocationOfInterest implements LocationOfInterest {
     readonly customId: string = '',
     readonly predefined: boolean = true
   ) {}
+
+  static getSmallestByArea(lois: List<LocationOfInterest>): LocationOfInterest {
```

Comment:
Is the diff just getting confused, and this is actually the same method in same class? Otherwise I would ask if this is getting tested.

Reply:
This is basically a merged version of LocationOfInterest and GenericLocationOfInterest. We're punting on tests on web for now due to lack of developer coverage, but we'll pick up adding coverage once the community project is ramped up.